hadoop-hdfs-project/hadoop-hdfs/src/test/java/io/hops/security/TestUsersGroups.java

```diff
@@ -425,79 +321,124 @@ public void testGetUnixUserGroups() throws Exception {
     HdfsStorageFactory.resetDALInitialized();
     HdfsStorageFactory.setConfiguration(conf);
     HdfsStorageFactory.formatStorage();
+    UsersGroups.createSyncRow();
```

Comment:
Isn't supposed to be called in UsersGroups.init?

Reply:
multi NN unit tests causes problem. it is static object.